Flutter/DateRangePicker/customizations.md

```diff
@@ -66,6 +66,9 @@ Widget build(BuildContext context) {
 
 ![Customizations Date Range Picker](images/customizations/customizations.png)
 
+>**NOTE**
+* Cell decoration will be rendered in this order blackout dates, special dates, disable dates, today date, weekend dates, trailing and leading dates and dates decoration based on the given decoration.
```

Comment:
Based on the given decoration, cell decoration will be rendered in the following order: blackout dates, special dates, disable dates, today date, weekend dates, trailing and leading dates, and dates decoration.

Reply:
I have changed and updated this in my latest commit